models/engine/db_storage.py

```diff
@@ -73,7 +73,7 @@ def reload(self):
         self.__session = Session
 
     def get(self, cls, id):
-        """A method to retrieve one object"""
+        """A method to retrieve one """
```

Comment:
When you are commenting the best option is to describe the return of the function, in this case, the object